cmd/node/config/external.toml

```diff
@@ -47,3 +47,10 @@
     RouteSendData = "/block"
     # Route used to acknowledge sent blocks
     RouteAcknowledgeData = "/acknowledge"
+
+[WebSocketsConnector]
+    # This flag shall only be used for observer nodes
+    Enabled = false
+    URL = "localhost:22111"
+    WithAcknowledge = true
+    MarshallerType = "json"
```

Comment:
document this config value. could be like
`# Currently, only "json" is supported. In the future, "gogo protobuf" could also be supported`

Reply:
done.